packages/n4s/src/enforce/compounds/anyOf.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * @param {*} value   Value to be test against rules
+ * @param {Function[]} rules    Rules to validate the value with
+ */
+export default function anyOf(value, ...rules) {
+  return !(rules.length) || rules.some(fn => fn(value));
```

Comment:
As I described in my comment, the `fn` might actually be a group of functions. It can be a chain of `enforce.isNumber().greaterThan(5)`... So just calling the function as is won't work. You're going to need to call the `test` function. I added a helper function that will help with that: `runLazyRules`. This is how you can fix that:
```suggestion
  return !rules.length || rules.some(ruleGroup => runLazyRules(ruleGroup, value));
```